server_admin/topics/sso-protocols/oidc.adoc

```diff
@@ -97,6 +97,80 @@ This is used by clients running on internet-connected devices that have limited
 . The application provides the user with the user code and the verification URI. The user accesses a verification URI to be authenticated by using another browser.
 . The application repeatedly polls {project_name} to find out if the user completed the user authorization. If user authentication is complete, the application exchanges the device code for an _identity_, _access_ and _refresh_ token.
 
+===== Client Initiated Backchannel Authentication Grant
+
+:tech_feature_name: Client Initiated Backchannel Authentication Grant
+:tech_feature_setting: -Dkeycloak.profile.feature.ciba=enabled
+include::../templates/techpreview.adoc[]
+
+This is used by clients who want to initiate the authentication flow by communicating with the OpenID Provider directly without redirect through the user's browser like OAuth 2.0's authorization code grant. Here's a brief summary of the protocol:
+
+. The client requests {project_name} an auth_req_id that identifies the authentication request made by the client. {project_name} creates the auth_req_id.
```

Comment:
As I mentioned in previous comments, it will be nice to add some "cross-linking" here as well and add links to other 2 places in the securing app chapters `Backchannel Authentication Endpoint` and `Client Initiated Backchannel Authentication Grant` .

Reply:
I see, I would like to add such a cross-linking sentence.

Reply:
+1, Thanks!